### PR TITLE
Prado support

### PR DESCRIPTION
**Pull request by alfredodeza**
Prado needs these files at the top level because they depend on the directory structure relative to their location which doesn't work if they exist only on `examples/`

It will make it easier for deploying prado.

### CODE REVIEW

File: ansible/examples/slave.yml [hunk 1]
```diff
@@ -194,28 +193,23 @@
...
-      failed_when: false
 
-    - name: update the etc cloud templates for debian /etc/hosts
+    - name: update etc cloud templates for red hat /etc/hosts
       sudo: true
       lineinfile:
+        backup: yes
         dest: /etc/cloud/templates/hosts.redhat.tmpl
         line: '127.0.1.1 ceph-builders'
-      failed_when: false
```
Comment (andrewschoen) [inline ansible/examples/slave.yml]: Why did you remove ``failed_when: false`` from these last two tasks? Those template files won't exist on all slaves and I think ``lineinfile`` will fail if the ``dest`` doesn't exist.

File: ansible/examples/slave_libvirt.yml [hunk 2]
```diff
@@ -74,7 +73,7 @@
       become_user: "{{ jenkins_user }}"
 
     - name: set the authorized keys
-      authorized_key: user={{ jenkins_user }} key="{{ lookup('file', 'files/ssh/keys/jenkins_build.pub') }}"
+      authorized_key: user={{ jenkins_user }} key="{{ lookup('file', 'playbook/files/ssh/keys/jenkins_build.pub') }}"
```
Comment (andrewschoen) [inline ansible/examples/slave_libvirt.yml]: Why was ``playbook`` added here? Is this something for ``prado`` support?

Comment (alfredodeza) [inline ansible/examples/slave_libvirt.yml]: ah this might be specific to prado then, I will add it back to examples (but not to the top level yaml files)

File: ansible/examples/slave.yml (hunk 1 above)
Comment (alfredodeza) [inline ansible/examples/slave.yml]: that is how this file currently exists in prado though
